src/components/Upgrade/content.js

```diff
@@ -22,95 +26,139 @@ export const highlights = [
   {
     title: {
       small: null,
-      large: 'Custom labels for apps and addresses',
+      large: 'Streamlined  setup flow',
     },
     description: {
       small: null,
       large: `
-        Give addresses and apps their own custom labels so you can easily stay
-        organized and navigate your organization.
+      Featuring more templates to match with your
+      needs. Start a company, membership, or
+      reputation-based organization in just a few
+      clicks.
```

Comment:
Nitpick: what do you think about indenting the strings here by one level, as they were previously?